pkg/cvo/cvo.go

```diff
@@ -238,17 +242,21 @@ func (optr *Operator) InitializeFromPayload(restConfig *rest.Config, burstRestCo
 	}
 	// XXX: set this to the cincinnati version in preference
 	if _, err := semver.Parse(update.ImageRef.Name); err != nil {
-		return fmt.Errorf("The local release contents name %q is not a valid semantic version - no current version will be reported: %v", update.ImageRef.Name, err)
+		return fmt.Errorf("the local release contents name %q is not a valid semantic version - no current version will be reported: %v", update.ImageRef.Name, err)
```

Comment:
This PR is epic.  Can we pull things like this and [the collector pointer bit][1] out into separate pull requests so they can land while the API refactor and feature extension stuff gets hashed out?

[1]: https://github.com/openshift/cluster-version-operator/pull/279#discussion_r356662055